<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="janjira" ignored="0" name="Test" tests="4" failures="1" timestamp="2566-03-04T13:54:10 ICT" time="0.098" errors="0">
  <testcase name="aTest3" time="0.007" classname="testNG.GroupDemoTest"/>
  <testcase name="aTest4" time="0.002" classname="testNG.GroupDemoTest"/>
  <testcase name="aTest1" time="0.002" classname="testNG.GroupDemoTest"/>
  <testcase name="aTest2" time="0.021" classname="testNG.GroupDemoTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testNG.GroupDemoTest.aTest2(GroupDemoTest.java:48)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- aTest2 -->
</testsuite> <!-- Test -->
